### PR TITLE
[C-981] Add suggested follows to feed if empty

### PR DESCRIPTION
**Pull request by raymondjacobson**
### Description

Add suggested follows to feed if it's empty.

Simulator Screen Shot - iPhone 14 Pro - 2022-11-04 at 18 44 13

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

Confirmed works on mobile & web!

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-981

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-981

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-981

File: packages/mobile/src/screens/feed-screen/FeedScreen.tsx [hunk 1]
```diff
@@ -55,6 +73,14 @@ export const FeedScreen = () => {
           pullToRefresh
           delineate
           selfLoad
+          LineupEmptyComponent={
+            <View style={{ height: dimensions.height - 200 }}>
+              <SuggestedFollows
```
Comment (dylanjeffers) [inline packages/mobile/src/screens/feed-screen/FeedScreen.tsx:78]: Sweet

File: packages/mobile/src/components/lineup/Lineup.tsx [hunk 2]
```diff
@@ -514,9 +518,10 @@ export const Lineup = ({
         onScroll={handleScroll}
         ListHeaderComponent={header}
         ListFooterComponent={<View style={{ height: 16 }} />}
+        ListEmptyComponent={LineupEmptyComponent}
         onEndReached={handleLoadMore}
         onEndReachedThreshold={LOAD_MORE_THRESHOLD}
-        sections={sections}
+        sections={areSectionsEmpty(sections) ? [] : sections}
```
Comment (dylanjeffers) [inline packages/mobile/src/components/lineup/Lineup.tsx:524]: Interesting, sg

**Review by dylanjeffers — APPROVED** [review]
Cool it could be reused like this

File: packages/mobile/src/components/lineup/Lineup.tsx (hunk 2 above)
Comment (raymondjacobson) [reply packages/mobile/src/components/lineup/Lineup.tsx:524]: Yeah this was somewhat annoying, but section list doesn't handle this by ddefault L:;(

Comment (raymondjacobson) [reply packages/mobile/src/components/lineup/Lineup.tsx:524]: If it has sections, even if their all empty, it thinks there's stuff

Comment (dylanjeffers) [reply packages/mobile/src/components/lineup/Lineup.tsx:524]: Ahh okay that makes a lot of sense actually consider it doesn't handle that by default

Comment (dylanjeffers) [reply packages/mobile/src/components/lineup/Lineup.tsx:524]: Can we add that to section list then?

Comment (raymondjacobson) [reply packages/mobile/src/components/lineup/Lineup.tsx:524]: Oh yeah I forgot we had our own. sure thing bossman

Comment (raymondjacobson) [reply packages/mobile/src/components/lineup/Lineup.tsx:521]: DONEZO

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-981

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/rj-c-981

**Pull request merged**